joomla/joomla-cms
# [4.0] Rephrase SEND_FAILURE_BODY when registration fails

Pull Request for Issue #33271 .

### Summary of Changes
A boolean field `$return` was being passed to `SEND_FAILURE_BODY` and it was displayed in the email body as shown below. This has been fixed by removing the line 'The error is: %s' and by not passing $return if the mail fails


### Testing Instructions
1. Enable Registrations for user by checking the allow registrations box that can be found in Admin Panel->Users->Options
![image](https://user-images.githubusercontent.com/53610833/115949709-8416e300-a4f4-11eb-83b4-241ca50e82f0.png)
2. Make sure you haven't set up an email for the mailer
3. Frontend -> Under the login panel module, select create a new account
4. Enter valid information and submit the form
5. Check Admin Panel -> Users -> Messaging -> Private Messages


### Actual result BEFORE applying this Pull Request
![image](https://user-images.githubusercontent.com/53610833/115949635-0652d780-a4f4-11eb-847c-0d5083ca7ea2.png)


### Expected result AFTER applying this Pull Request
![image](https://user-images.githubusercontent.com/53610833/115949624-fd620600-a4f3-11eb-90cc-7e47b4b0811f.png)


### Documentation Changes Required
None
